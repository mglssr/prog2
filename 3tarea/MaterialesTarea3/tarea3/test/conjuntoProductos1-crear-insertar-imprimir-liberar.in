crearConjuntoProductos 2
imprimirConjuntoProductos
liberarConjuntoProductos
crearConjuntoProductos 2
insertarConjuntoProductos 1
insertarConjuntoProductos 2
insertarConjuntoProductos 1
imprimirConjuntoProductos
liberarConjuntoProductos
Fin
