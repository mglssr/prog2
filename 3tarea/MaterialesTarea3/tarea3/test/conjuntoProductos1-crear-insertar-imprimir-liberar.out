Bienvenido al programa principal de MercadoFinger. Por favor ingrese su comando:
1> El conjunto productos fue creado en forma correcta.
2> Conjunto de productos:

3> Conjunto de productos liberado exitosamente.
4> El conjunto productos fue creado en forma correcta.
5> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
6> Se ejecuto 'insertarTConjuntoProductos' con producto de id 2 exitosamente.
7> Se ejecuto 'insertarTConjuntoProductos' con producto de id 1 exitosamente.
8> Conjunto de productos:
1 
9> Conjunto de productos liberado exitosamente.
10> Fin.
